nouveau-app/src/routes/Registration.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import {useState} from 'react';
+import ReactDOM from 'react-dom/client';
+import {Button, Col, Form, FormGroup, Input, Label, Row} from 'reactstrap';
+import axios from "axios";
+
+// password requirements?
+// encryption?
+
+
+let REGISTRATION_URL = "http://localhost:8080/registration";
```

Comment:
I asked this for my backend PR too, but is there a way to set up environment variables in Azure so we don't have to change this between development and deployment?